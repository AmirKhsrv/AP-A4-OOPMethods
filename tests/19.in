best_book
show_author_info 1534
best_reviewer
best_reviewer
recommend_first_approach 95
recommend_second_approach 27
credit 16
credit 43
show_sorted_shelf 31 currently_reading Fantasy
best_book
