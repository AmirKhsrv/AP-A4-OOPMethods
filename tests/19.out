id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 1534
Name: Anne Perry
Year of Birth: 1994
Place of Birth: Honduras
Member Since: July 5 2011
Genres: Fiction
Books:
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 392
Title: From the Mixed-Up Files of Mrs. Basil E. Frankweiler
Genre: Fiction
Author: E.L. Konigsburg
Reviews:
id: 356 Rating: 3 Likes: 0 Date: August 24 2018
id: 396 Rating: 2 Likes: 558 Date: January 16 2011
id: 677 Rating: 5 Likes: 3 Date: August 13 2018
id: 914 Rating: 5 Likes: 0 Date: June 3 2019
id: 1654 Rating: 0 Likes: 0 Date: February 6 2019
id: 1665 Rating: 5 Likes: 0 Date: November 21 2017
id: 1866 Rating: 0 Likes: 0 Date: December 27 2018
id: 2173 Rating: 2 Likes: 0 Date: February 17 2012
id: 2285 Rating: 5 Likes: 0 Date: March 28 2013
id: 2901 Rating: 5 Likes: 0 Date: July 20 2017
id: 4481 Rating: 5 Likes: 2 Date: May 21 2016
id: 5827 Rating: 4 Likes: 0 Date: November 25 2019
id: 5975 Rating: 1 Likes: 0 Date: March 28 2018
id: 6899 Rating: 5 Likes: 1 Date: February 27 2018
id: 8914 Rating: 1 Likes: 0 Date: July 12 2013
id: 9248 Rating: 5 Likes: 0 Date: October 4 2018
id: 9999 Rating: 5 Likes: 0 Date: March 18 2015
id: 10317 Rating: 4 Likes: 0 Date: April 18 2014
id: 10521 Rating: 4 Likes: 0 Date: September 13 2019
id: 10822 Rating: 5 Likes: 0 Date: May 11 2016
id: 11689 Rating: 5 Likes: 1 Date: January 27 2017
id: 12274 Rating: 5 Likes: 0 Date: August 25 2015
id: 12379 Rating: 5 Likes: 0 Date: June 23 2015
id: 12431 Rating: 0 Likes: 1 Date: July 25 2019
id: 12626 Rating: 3 Likes: 0 Date: September 18 2019
id: 13444 Rating: 0 Likes: 0 Date: June 26 2013
id: 13813 Rating: 3 Likes: 0 Date: May 27 2018
id: 14083 Rating: 3 Likes: 0 Date: October 26 2014
id: 14142 Rating: 4 Likes: 0 Date: December 27 2014
id: 14199 Rating: 3 Likes: 4 Date: October 12 2015
id: 14427 Rating: 1 Likes: 2 Date: June 15 2018
id: 16431 Rating: 5 Likes: 0 Date: October 28 2014
id: 16509 Rating: 5 Likes: 0 Date: March 17 2016
id: 16740 Rating: 3 Likes: 0 Date: April 26 2016
id: 16829 Rating: 0 Likes: 0 Date: October 15 2017
id: 17340 Rating: 3 Likes: 0 Date: November 13 2017
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
Reviews:
id: 99 Rating: 3 Likes: 8 Date: November 16 2018
id: 142 Rating: 4 Likes: 0 Date: January 20 2019
id: 949 Rating: 1 Likes: 1 Date: May 14 2017
id: 1209 Rating: 1 Likes: 0 Date: August 22 2013
id: 1350 Rating: 5 Likes: 2 Date: September 25 2018
id: 1595 Rating: 1 Likes: 0 Date: July 1 2013
id: 1747 Rating: 1 Likes: 0 Date: November 13 2018
id: 1846 Rating: 4 Likes: 0 Date: July 9 2016
id: 3401 Rating: 3 Likes: 0 Date: January 1 2016
id: 3889 Rating: 5 Likes: 0 Date: October 17 2017
id: 4887 Rating: 5 Likes: 3 Date: May 4 2018
id: 5127 Rating: 5 Likes: 0 Date: November 9 2010
id: 5574 Rating: 3 Likes: 0 Date: October 6 2014
id: 6137 Rating: 3 Likes: 0 Date: June 28 2017
id: 6480 Rating: 1 Likes: 1 Date: May 1 2011
id: 7559 Rating: 0 Likes: 7 Date: August 19 2017
id: 7913 Rating: 3 Likes: 0 Date: July 10 2019
id: 8048 Rating: 2 Likes: 0 Date: December 23 2018
id: 8160 Rating: 1 Likes: 0 Date: February 26 2017
id: 8310 Rating: 2 Likes: 0 Date: November 29 2019
id: 8626 Rating: 5 Likes: 0 Date: June 16 2014
id: 8683 Rating: 4 Likes: 0 Date: June 20 2013
id: 8858 Rating: 0 Likes: 0 Date: December 16 2015
id: 8862 Rating: 4 Likes: 0 Date: August 26 2018
id: 8903 Rating: 5 Likes: 0 Date: July 11 2018
id: 9182 Rating: 2 Likes: 0 Date: October 12 2018
id: 10427 Rating: 2 Likes: 0 Date: July 28 2018
id: 10432 Rating: 3 Likes: 0 Date: January 9 2018
id: 10490 Rating: 4 Likes: 0 Date: October 20 2017
id: 11047 Rating: 2 Likes: 24 Date: October 15 2010
id: 12102 Rating: 2 Likes: 3 Date: April 22 2018
id: 12147 Rating: 4 Likes: 2 Date: December 12 2019
id: 12633 Rating: 3 Likes: 0 Date: March 21 2016
id: 13302 Rating: 4 Likes: 0 Date: April 18 2015
id: 14111 Rating: 3 Likes: 1 Date: September 14 2016
id: 14698 Rating: 2 Likes: 0 Date: October 22 2019
id: 16634 Rating: 2 Likes: 0 Date: October 13 2018
0.003615
0.003896
id: 329
Title: Alias Vol. 1
Genre: Sequential Art
Author: Brian Michael Bendis
***
id: 230
Title: Fables Vol. 1: Legends in Exile
Genre: Sequential Art
Author: Bill Willingham
***
id: 190
Title: How to Win Friends and Influence People
Genre: Nonfiction
Author: Dale Carnegie
***
id: 219
Title: Strands of Bronze and Gold
Genre: Young Adult
Author: Jane Nickerson
***
id: 133
Title: The Strange and Beautiful Sorrows of Ava Lavender
Genre: Young Adult
Author: Leslye Walton
***
id: 23
Title: You're Welcome Universe
Genre: Young Adult
Author: Whitney Gardner
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
